### PR TITLE
XRT-2585: Updated canbus v3.0 examples

### PR DESCRIPTION
**Pull request by Tylerg2471**

### CODE REVIEW

File: DeviceServices/canbus/deployment/config/2-logger.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "Name":"console",
+  "Level":"Trace"
```
Comment (BinduRao2018) [inline DeviceServices/canbus/deployment/config/2-logger.json]: Should this be 'Warn'?

**Review by BinduRao2018 — COMMENTED** [review]
@Tylerg2471, one minor comment

File: DeviceServices/canbus/deployment/config/2-logger.json (hunk 1 above)
Comment (SteveOss) [reply DeviceServices/canbus/deployment/config/2-logger.json]: I would say the the default log level for all examples should perhaps info or warn. Thoughts ?

Comment (Tylerg2471) [reply DeviceServices/canbus/deployment/config/2-logger.json]: Yeah Warn is what it normally is set to will fix.

File: DeviceServices/canbus/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# XRT Config Examples
+
+## Overview
+
+This page shows you how to setup and run the CANbus device service example.
+
+For more information about the Device Service please review the [CANbus Device Service](hhttps://docs.iotechsys.com/edge-xrt20/device-service-components/canbus-device-service-component.html) documentation.
```
Comment (SteveOss) [inline DeviceServices/canbus/README.md]: Should be a link to v3 not v2

**Review by SteveOss — CHANGES_REQUESTED** [review]

**Review by SteveOss — CHANGES_REQUESTED** [review]
Does the link actually resolve to anything ?

**Comment by Tylerg2471** [comment]
> Does the link actually resolve to anything ?

No it doesn't, where are the 3.0 connect docs hosted?

**Comment by SteveOss** [comment]
Look under: https://docs.iotechsys.com/edge-connect30/

**Review by BinduRao2018 — APPROVED** [review]
LGTM

**Comment by BinduRao2018** [comment]
@SteveOss, The 3.0 doc isn't hosted yet, but the links are now pointing to them. Is this ok?

**Comment by BinduRao2018** [comment]
@SteveOss Is this good to go?

**Review by SteveOss — APPROVED** [review]
LGTM

**Pull request merged**